JS1064_JS104_terminate_a_loop/script.js

```diff
@@ -0,0 +1,12 @@
+let message = "supercalifragilisticexpialidocious";
+message = message.length;
+
+for ( let i = 1; i < message; i++ ) {
+  if ( i === message / 2 ) {
+    console.log('The loop has terminated...');
+    break;
+  }
+  console.log(`Logging the number ${i}`);
+}
+
+console.log('The program continues...');
```

Comment:
This will work, but this can be improved to this
```suggestion
const message = "supercalifragilisticexpialidocious";

for ( let i = 1; i < message.length; i++ ) {
  if ( i === message.length / 2 ) {
    console.log('The loop has terminated...');
    break;
  }
  console.log(`Logging the number ${i}`);
}

console.log('The program continues...');
```
This improves the readability since we are not transposing the `message` value from text to string length. Will approve since this was the sample from the training track.